sections/model_overview/numerical_results/main.tex

```diff
@@ -293,81 +226,81 @@ \subsection{Numerical results}\label{sec:numerical_results}
 \(\tilde{B}\) of matrix \(B\) only on the column that is most frequently played
 (matrix \(A\): row \(5\), matrix \(B\): column \(6\), see Figure 
 \ref{fig:ard_penalty}).
-Note that the values of the payoff matrices only differ on the fourth digit
-which is the raw utility of the strategies played.
+Note that for the presentation of data, a scaling has been applied to obtain
+the values of the payoff matrices.
+The results are not affected by this scaling.
```

Comment:
```suggestion
Note that for the presentation of data, an affine transformation has been applied to obtain
the values of the payoff matrices.
The results are not affected by this scaling.
```

Reply:
We should specifically write what the transformation is (in brackets somewhere, it's not super important).